learning_resources/models.py

```diff
@@ -233,15 +240,52 @@ class Program(TimestampedModel):
         on_delete=models.deletion.CASCADE,
         primary_key=True,
     )
-    courses = models.ManyToManyField(
-        LearningResource,
-        related_name="programs",
-        limit_choices_to={
-            "learning_resource__resource_type": LearningResourceType.course.value
-        },
-    )
 
     @property
     def runs(self):
         """Get the parent LearningResource runs"""
         return self.learning_resource.runs
+
+    @property
+    def courses(self):
+        """Get the associated resources (should all be courses)"""
+        return self.learning_resource.children
+
+
+class LearningPath(TimestampedModel):
+    """
+    Model for representing a publishable list of  learning resources
+    The LearningResource readable_id should probably be something like an auto-generated UUID.
+    """
+
+    learning_resource = models.OneToOneField(
+        LearningResource,
+        related_name="learning_path",
+        on_delete=models.CASCADE,
+    )
+    author = models.ForeignKey(
+        User, related_name="learning_paths", on_delete=models.PROTECT
+    )
+
+    def __str__(self):
+        return f"Staff List: {self.learning_resource.title}"
```

Comment:
IMO, keep `LearningPath` for the `__str__` name.

Reply:
>Given a resource, decide what lists it belongs to. E.g., for the 'add to list' dialog

The list it belongs to should be in the url, ie posting to http://localhost:8063/api/v1/learningpaths/55/resources/ will add it to the LearningResource with id 55.


> logic around updating positions on PATCH and DELETE requests. (e.g., deleting { position: 4 }` shifts 5+ down, PATCHing moves items toward old position)
For patching, that is in the LearningResourceRelationshipSerializer.update() function.  For delete, we could add that, though it should make no difference to the remaining items order, there will just be a numerical gap in the position values.

Reply:
https://github.com/mitodl/mit-open/issues/51